modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -21,10 +21,11 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. Optional: You can import a `Devfile`, a `Dockerfile`, or a `Builder Image` through your Git repository to further customize your deployment.
-* If your Git repository contains a `Devfile`, a `Dockerfile`, or a `Builder Image`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default.
+. Optional: You can import a `Devfile`, a `Dockerfile`, `Builder Image` or a `Serverless Function` through your Git repository to further customize your deployment.
+* If your Git repository contains a `Devfile`, a `Dockerfile`, `Builder Image`, or a `func.yaml`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default. If `func.yaml` is detected in the Git repository, the *Import Strategy* changes to `Serverless Function`.
```

Comment:
These appear to be different points, so maybe consider turning them into independent list items, rather than keep adding more independent sentences under one bullet point:
```suggestion
. Optional: You can import a `Devfile`, `Dockerfile`, `Builder Image` or a `Serverless Function` through your Git repository to further customize your deployment.
* If your Git repository contains a `Devfile`, a `Dockerfile`, `Builder Image`, or `func.yaml`, it is automatically detected and populated on the respective path fields.
* If a `Devfile`, `Dockerfile`, or `Builder Image` are detected in the same repository, the `Devfile` is selected by default.
* If `func.yaml` is detected in the Git repository, the *Import Strategy* changes to `Serverless Function`.
```

Reply:
Agreed! Made the changes. Thanks!